RELEASE_NOTES.md

```diff
@@ -8,6 +8,55 @@ autoconf   | >=2.61
 zlib1g-dev | >=0.13.x
 libtool    | >=2.x
 
-New Features
+Migration Instructions
+======================
+
+* The generated documentation is now written to a top-level folder `sphinx-build`.
+  Since `clean*` targets have been changed accordingly, documentation generated
+  prior to this release will have to be removed manually.
+
+Issues Fixed
 ============
 
+* Bugfix where calls to `drizzle_connect()` would always block regardless of the
+  `non_blocking` option being set to true.
+	([#41](https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/41))
+
+* Optimization of `drizzle_field_buffer()` so an array of `drizzle_field_t`
```

Comment:
No. It was the reason for having a summary at the top. 
Looked at different ways of writing the RELEASE_NOTES, and didn't find a definite guideline. 
Wanted to give a description of the central changes in the release, but patch releases are normally only a list of things fixed, it doesn't seem to fit the bil. 
Was considering changing it to a minor release, since the async binlog is sort of a new feature.